src/serlio/prtMaterial/ArnoldMaterialNode.cpp

```diff
@@ -0,0 +1,575 @@
+		size_t tmpSize = buf.size();
+		sb.setVar(L"$mapNode", shadingGroupName + L"_normal_map");
+		sb.setVar(L"$mapFile", prt::StringUtils::toUTF16FromOSNarrow(matInfo.normalMap.c_str(), buf.data(), &tmpSize));
+		sb.createTexture(L"$mapNode");
+		sb.setAttr(L"($mapNode + \".fileTextureName\")", L"$mapFile");
+		sb.setVar(L"$uvTrafoNode", shadingGroupName + L"_normal_map_trafo");
+		sb.createShader(L"aiUvTransform", L"$uvTrafoNode");
+		sb.setAttr(L"($uvTrafoNode + \".uvset\")", L"\"normalMap\"");
+		sb.setAttr(L"($uvTrafoNode + \".pivotFrame\")", 0.0, 0.0);
+		sb.setAttr(L"($uvTrafoNode + \".scaleFrame\")", 1.0 / matInfo.normalmapTrafo.su(), 1.0 / matInfo.normalmapTrafo.sv());
+		sb.setAttr(L"($uvTrafoNode + \".translateFrame\")", -matInfo.normalmapTrafo.tu() / matInfo.normalmapTrafo.su(), -matInfo.normalmapTrafo.tv() / matInfo.normalmapTrafo.sv());
+		if (matInfo.normalmapTrafo.rw() != 0.0) {
+			LOG_WRN << "rotation (material.map.rw) is not yet supported\n";
+		}
+
+		sb.connectAttr(L"($mapNode + \".outColor\")", L"($uvTrafoNode + \".passthrough\")");
+		sb.connectAttr(L"($uvTrafoNode + \".outColor\")", L"($shaderNode + \".normalCamera\")");
+	}
+
+	// emission
+	sb.setAttr(L"($shaderNode + \".emission\")", 1.0);
+
+	// emission/emissive map multiply node
+	sb.setVar(L"$emissiveMapBlendNode", shadingGroupName + L"_emissive_map_blend");
+	sb.createShader(L"aiMultiply", L"$emissiveMapBlendNode");
+	sb.connectAttr(L"($emissiveMapBlendNode + \".outColor\")", L"($shaderNode + \".emissionColor\")");
+
+	// emissive color
+	sb.setAttr(L"($emissiveMapBlendNode + \".input1\")", matInfo.emissiveColor);
+
+	// emissive map
+	if (matInfo.emissiveMap.empty()) {
+		sb.setAttr(L"($emissiveMapBlendNode + \".input2\")", 1.0, 1.0, 1.0);
+	}
+	else {
+		size_t tmpSize = buf.size();
+		sb.setVar(L"$mapNode", shadingGroupName + L"_emissive_map");
+		sb.setVar(L"$mapFile", prt::StringUtils::toUTF16FromOSNarrow(matInfo.emissiveMap.c_str(), buf.data(), &tmpSize));
+		sb.createTexture(L"$mapNode");
+		sb.setAttr(L"($mapNode + \".fileTextureName\")", L"$mapFile");
+		sb.setVar(L"$uvTrafoNode", shadingGroupName + L"_emissive_map_trafo");
+		sb.createShader(L"aiUvTransform", L"$uvTrafoNode");
+		sb.setAttr(L"($uvTrafoNode + \".uvset\")", L"\"emissiveMap\"");
+		sb.setAttr(L"($uvTrafoNode + \".pivotFrame\")", 0.0, 0.0);
+		sb.setAttr(L"($uvTrafoNode + \".scaleFrame\")", 1.0 / matInfo.emissivemapTrafo.su(), 1.0 / matInfo.emissivemapTrafo.sv());
+		sb.setAttr(L"($uvTrafoNode + \".translateFrame\")", -matInfo.emissivemapTrafo.tu() / matInfo.emissivemapTrafo.su(), -matInfo.emissivemapTrafo.tv() / matInfo.emissivemapTrafo.sv());
+		if (matInfo.emissivemapTrafo.rw() != 0.0) {
+			LOG_WRN << "rotation (material.map.rw) is not yet supported\n";
+		}
+
+		sb.connectAttr(L"($mapNode + \".outColor\")", L"($uvTrafoNode + \".passthrough\")");
+		sb.connectAttr(L"($uvTrafoNode + \".outColor\")", L"($emissiveMapBlendNode + \".input2\")");
+	}
+
+	// TODO: support occlusion map

```

Comment:
i guess, we do not need to do that with arnold beeing a raytracer

Reply:
yep, I thought as much